_gsocproposals/2018/proposal_ATLASEigen.md

```diff
@@ -2,6 +2,7 @@
 title:  Faster matrix algebra for ATLAS
 layout: gsoc_proposal
 project: ATLAS
+year: 2018
```

Comment:
Do we need the year metadata anymore? It seems like you now get this during processing from the path.

Reply:
If we decide we don't need it then it should be removed from the 2017 proposals and the instructions.

Reply:
If we fix the usage of page.year in _layouts/gsoc_proposal.html we should be able to drop the year from the metadata.